Plugins/BTCPayServer.Plugins.ShopifyPlugin/Services/ShopifyHostedService.cs

```diff
@@ -127,7 +131,7 @@ async Task<InvoiceLogs> Process(long shopifyOrderId, InvoiceEntity invoice)
                     await client.CaptureOrder(new()
                     {
                         Currency = invoice.Currency,
-                        Amount = invoice.Price,
+                        Amount = Math.Round(invoice.PaidAmount.Net, _currencyNameTable.GetNumberFormatInfo(invoice.Currency).CurrencyDecimalDigits),
```

Comment:
1. `GetNumberFormatInfo()` can returns `null`. In which case what shall we do? Maybe fallback to 2 decimal?
2. I think if the status is `Settled`, just use `invoice.Price`. This will issues related to strange rounding corner case. Also, it will avoid wondering what happen in case of over payment.